packages/boxed-expression-component/src/contextMenu/MenuWithHelp/MenuItemWithHelp.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import "./MenuWithHelp.css";
+import { MenuItem, MenuItemAction } from "@patternfly/react-core/dist/js/components/Menu";
+import { HelpIcon } from "@patternfly/react-icons/dist/js/icons/help-icon";
+
+export interface MenuItemWithHelpProps {
+  menuItemKey: string;
+  menuItemCustomText?: string;
+  menuItemDescription: string;
+  menuItemIcon?: any;
+  menuItemIconStyle?: any;
+  setVisibleDescription: (description: string) => void;
+  getVisibleDescription: () => string;
+}
+
+export function MenuItemWithHelp({
+  menuItemKey,
+  menuItemCustomText,
+  menuItemDescription,
+  menuItemIcon,
+  menuItemIconStyle,
+  setVisibleDescription,
+  getVisibleDescription,
+}: MenuItemWithHelpProps) {
+  const [isHoveredMenuItem, setIsHoveredMenuItem] = React.useState<boolean>();
```

Comment:
Since this is inside this component, you can rename this to `isHovered` simply.

Reply:
done in  a separate commit 8f50bd0b49